src/api/Pluribus - Account.postman_collection.json

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
Fixture data is OK to version, but it should live in a directory outside of the `src` directory.